packages/react-components/react-text/src/components/Text/Text.types.ts

```diff
@@ -14,9 +14,9 @@ export type TextProps = ComponentProps<TextSlots> & {
   /**
    * Aligns text based on the parent container.
    *
-   * @default start
+   * @default 'none'
```

Comment:
What do you think about using `inherit` for the default value to better align with CSS keywords?

https://developer.mozilla.org/en-US/docs/Web/CSS/inherit

Reply:
When a prop is nullable, the default can be for us to not set any style and leave them undefined.  This can avoid creating another value in the union.

Reply:
I like the idea of having the prop be `undefined` by default. `inherit` doesn't feel like quite the right name and the prop is already optional.

Reply:
I think `inherit` would create more confusion as we wouldn't actually apply that. I went with `none` to keep it consistent with our lib as I'm not aware of any component working in `undefined` values, as we're forcing `Required` on component state.
I'm pro using `undefined` but it needs to be coherent and standardized.

Reply:
IMO folks meant following:

```diff
-align?: 'none' | 'start' | 'center' | 'end' | 'justify';
+align?: 'start' | 'center' | 'end' | 'justify';
```

optional operator ads undefined type already

Reply:
Right, but that would be for `TextProps`. Users don't need to define the prop value, but all `*State` types use Required to force all optional props to be required.
I'm pro not having this behavior (as I think it doesn't make sense and have previously voiced my opinion), but this is not coherent with the rest of the library.

Reply:
At this point, we should opt for API consistency above all. I'm open to other values to represent no action other than `none` however.

Reply:
`unset` is another good option that matches CSS. The [CSS keyword unset](https://developer.mozilla.org/en-US/docs/Web/CSS/unset) represents the default value and is equivalent to not setting the property at all.

Reply:
don't think `unset` could be used as that would go against implementation intent. we dont wanna do actual "reset" of css style definition.

Reply:
we had additional offline sync with @andrefcdias on this - we will create a draft RFC for alignment across all component APIs on how to approach "default" values. once that will be resolved we can go back and re-solve this convo accordingly. ty!

Reply:
RFC: https://github.com/microsoft/fluentui/pull/23304

Reply:
`undefined` is fine with me! 

To clarify, `unset` means "not set by the component" rather than "reset" (aka `initial`). I felt that `none` implies actual "reset" which is why I've been asking for a name change :)